entrypoint.sh

```diff
@@ -48,9 +59,15 @@ if [ -z "$TV_NAME" ]; then
 fi
 echo "Found TV name: $TV_NAME"
 
-echo "Attempting to install jellyfin-tizen-builds $JELLYFIN_BUILD_OPTION.wgt from release: $TAG"
-echo "$DOWNLOAD_URL"
+echo "Downloading jellyfin-tizen-builds $JELLYFIN_BUILD_OPTION.wgt from release: $TAG"
+wget -q --show-progress "$DOWNLOAD_URL"; echo ""
 
-wget -q --show-progress "$DOWNLOAD_URL"
+if ! [ -z "$CERTIFICATE_PASSWORD" ]; then
+	echo "Attempting to sign package using provided certificate"
+	sed -i "s/_CERTIFICATEPASSWORD_/$CERTIFICATE_PASSWORD/" profile.xml
+	sed -i '/<\/profile>/ r profile.xml' /home/developer/tizen-studio-data/profile/profiles.xml
+	tizen package -t wgt -s custom -- $JELLYFIN_BUILD_OPTION.wgt
+fi
 
+echo "Attempting to install jellyfin-tizen-builds $JELLYFIN_BUILD_OPTION.wgt from release: $TAG"
 tizen install -n $JELLYFIN_BUILD_OPTION.wgt -t "$TV_NAME"
```

Comment:
could you not automate the certificate automation using the tizen `certificate` command? https://developer.tizen.org/development/tizen-studio/web-tools/cli#Issue_tizen_cert

I'm trying something like this at the end of `entrypoint.sh`:
```shell
tizen certificate -a jellyfin -p 1234
tizen package -t wgt -s jellyfin -- $JELLYFIN_BUILD_OPTION.wgt
tizen install -n $JELLYFIN_BUILD_OPTION.wgt -t "$TV_NAME"

```
but still not working correctly. But I'm hoping someone smarter than me sees this and can understand why

Reply:
You’ve got to create a Samsung Certificate assigned to the DUID of your tv using Samsung Certificate Manager. The Tizen certificate no longer works.  There’s a few guides in the original ticket https://github.com/Georift/install-jellyfin-tizen/issues/21.